src/inference/src/streams_executor.hpp

```diff
@@ -50,6 +50,12 @@ void parse_processor_info_linux(const int _processors,
                                 int& _cores,
                                 std::vector<std::vector<int>>& _proc_type_table,
                                 std::vector<std::vector<int>>& _cpu_mapping_table);
+
+void get_cpu_mapping_from_cores(const int _processors,
```

Comment:
The function should have signature and behavior disciption (same as for others)

Reply:
Ok, added the description.